shell/platform/android/io/flutter/embedding/engine/dynamicfeatures/PlayStoreDynamicFeatureManager.java

```diff
@@ -249,6 +287,20 @@ public void downloadDynamicFeature(int loadingUnitId, String moduleName) {
             });
   }
 
+  public String getDynamicFeatureInstallState(int loadingUnitId, String moduleName) {
+    String resolvedModuleName =
+        moduleName != null ? moduleName : loadingUnitIdToModuleName(loadingUnitId);
+    if (resolvedModuleName == null) {
+      Log.d(TAG, "Dynamic feature module name was null.");
```

Comment:
This is probably a more prominent error right? If they look up something that don't exist, don't silently fail.